mods/seasonal/moves.js

```diff
@@ -1515,8 +1521,8 @@ let BattleMovedex = {
 		accuracy: 100,
 		basePower: 40,
 		category: "Special",
-		desc: "",
-		shortDesc: "",
+		desc: "Has a 100% chance to flinch the target. If this move hits a Pokemon not behind a Substitute, and if both the user and target are not already transformed or under the effect of Illusion, the user transforms into the opponent, copying its current stats, stat stages, types, moves, Ability, weight, gender, and sprite. The user's level and HP remain the same and each copied move receives only 5 PP, with a maximum of 5 PP each. The user can no longer change formes if it would have the ability to do so.",
+		shortDesc: "Hits first; user flinches and transforms into target.",
```

Comment:
target flinches, not the user

Reply:
the intent is to distribute "target" to both "flinches" and "transforms into". If it's not reading like that, though, it should be changed: "Hits first; user flinches target and transforms into it" assuming that fits?

Reply:
Ohhhh, now i see it. nvm

Reply:
Its confusing, I'd go with `Hits first; foe flinches; user transforms into target`

Reply:
It does, and did. Also, I'll probably add the new ones soon.